compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -17,13 +17,149 @@
 #include "luci/Pass/ResolveCustomOpAddPass.h"
 
 #include <gtest/gtest.h>
+#include <luci/test/TestIOGraph.h>
+
+#include <luci/IR/CircleNodes.h>
+
+using namespace luci::test;
 
 namespace
 {
 
+/**
+ *   Test graph with Custom(AddV2) to resolve
+ *
+ * [Pattern 1]
+ *        [Input]   [BroadcastTo]
+ *             \         /
+ *           [Custom(AddV2)]
+ *                  |
+ *             [CustomOut]
+ *                  |
+ *               [Output]
+ *
+ * [Pattern 2]
+ *        [Input]   [Custom(BroadcastTo)]
+ *             \         /
+ *           [Custom(AddV2)]
+ *                  |
+ *             [CustomOut]
+ *                  |
+ *               [Output]
+ */
+class BroadcastToAddGraphlet
+{
+public:
+  BroadcastToAddGraphlet() = default;
+
+public:
+  void init(loco::Graph *g, bool *isCustomBroadcastTo)
+  {
+    // CircleCustom(AddV2)
+    _addV2 = g->nodes()->create<luci::CircleCustom>(2, 1);
+    _addV2->custom_code("AddV2");
+    _addV2->shape({2, 3});
+    _addV2->dtype(loco::DataType::FLOAT32);
+    _addV2->name("addV2");
+
+    // CircleConst (BroadcastTo-input)
+    auto input = g->nodes()->create<luci::CircleConst>();
+    input->dtype(loco::DataType::FLOAT32);
+    input->shape({1, 3});
+    input->size<loco::DataType::FLOAT32>(3);
+    input->at<loco::DataType::FLOAT32>(0) = 1;
+    input->at<loco::DataType::FLOAT32>(1) = 2;
+    input->at<loco::DataType::FLOAT32>(2) = 3;
+
+    // CircleConst (BroadcastTo-shape)
+    auto shape = g->nodes()->create<luci::CircleConst>();
+    shape->dtype(loco::DataType::S32);
+    shape->shape({2});
+    shape->size<loco::DataType::S32>(2);
+    shape->at<loco::DataType::S32>(0) = 2;
+    shape->at<loco::DataType::S32>(1) = 3;
+
+    if (isCustomBroadcastTo)
+    {
```

Comment:
I mean put these into `BroadcastToAddV2Graph::init()`

Reply:
I revised it. Thank you a lot :)